library/src/main/java/com/novoda/downloadmanager/DownloadBatchIdCreator.java

```diff
@@ -6,7 +6,12 @@ private DownloadBatchIdCreator() {
         // non-instantiable class
     }
 
-    public static DownloadBatchId createFrom(String rawId) {
-        return new LiteDownloadBatchId(rawId);
+    public static DownloadBatchId createSanitizedFrom(String rawId) {
```

Comment:
add Javadoc with what this does to the param

Reply:
604b41f